platform/ui/src/components/SplitButton/SplitButton.tsx

```diff
@@ -26,15 +26,15 @@ const classes = {
       baseClasses.Button,
       !isExpanded &&
         !primary.isActive &&
-        'hover:bg-primary-dark hover:border-primary-dark'
+        'hover:!bg-primary-dark hover:border-primary-dark'
```

Comment:
Many buttons were affected by the upgrade to tailwind. To be honest, this actually demonstrates how fragile the look-and-feel implementation is.